src/topology/continuous_function/compact.lean

```diff
@@ -238,6 +238,13 @@ def linear_isometry_bounded_of_compact :
   norm_map' := λ f, rfl,
   .. add_equiv_bounded_of_compact α E }
 
+variables {α E} -- to match bounded_continuous_function.eval_clm
```

Comment:
can you switch back to the original setting after the lemma?

Reply:
I thought the "variables" declaration is local to the section, which ends immediately after the new lemma. So restating `variables (α E)` would be a no-op, wouldn't it?

Reply:
Sorry, I hadn't seen the `end`. You're absolutely right.